test/health_check_test_case.py

```diff
@@ -84,6 +89,35 @@ def test_health_endpoint_keys(self):
                         self.assertEqual(200, response.status_code)
                         self.assertEqual(expected_response, response.json())
 
+    @mock_s3
+    @mock_sts
+    @mock_sqs
+    def test_cached_health(self):
+        storage_service = StorageService()
+        storage_service.create_bucket()
+        with ResponsesHelper() as helper:
+            helper.add_passthru(self.base_url)
+            response = requests.get(self.base_url + '/health/cached')
+            self.assertEqual(500, response.status_code)
+
+        self._create_mock_queues()
+        endpoint_states = self._make_endpoint_states(self.endpoints)
+        app = load_app_module(self.lambda_name())
+        with ResponsesHelper() as helper:
+            helper.add_passthru(self.base_url)
+            self._mock_service_endpoints(helper, endpoint_states)
+            with mock.patch.dict(os.environ, AWS_DEFAULT_REGION='us-east-1'):
```

Comment:
The need for this isn't immediately apparent to me. Can you explain in a comment? Or, if it should be ovbious, let me know by replying to this comment.

Reply:
Checkout: https://api.travis-ci.org/v3/job/587629946/log.txt
The patch is required for all the health test that access `api_endpoints`, in most cases test that involve the service.

Reply:
I see. I'm glad I asked. In that case, let's make _mock_service_endpoints an @contextmanager and move the region patching into that method …

Reply:
Also, it looks as if `boto3.resource('sqs')` is the culprit that requires the region to be set. In _mock_service_endpoints I would mention that fact so the next person isn't puzzled like I was.